ecl/resource_ecl_mlb_load_balancer_v1.go

```diff
@@ -367,10 +391,14 @@ func resourceMLBLoadBalancerV1UpdateConfigurations(d *schema.ResourceData, clien
 			isConfigurationsUpdated = true
 
 			for i, interfaceV := range d.Get("interfaces").([]interface{}) {
-				results := make([]load_balancers.CreateStagedOptsReservedFixedIP, len(interfaceV.(map[string]interface{})["reserved_fixed_ips"].([]interface{})))
-				for j, reservedFixedIP := range interfaceV.(map[string]interface{})["reserved_fixed_ips"].([]interface{}) {
-					results[j] = load_balancers.CreateStagedOptsReservedFixedIP{
-						IPAddress: reservedFixedIP.(map[string]interface{})["ip_address"].(string),
+				results := []load_balancers.CreateStagedOptsReservedFixedIP{}
+				if reservedFixedIPsRaw, ok := interfaceV.(map[string]interface{})["reserved_fixed_ips"].([]interface{}); ok {
+					results = make([]load_balancers.CreateStagedOptsReservedFixedIP, len(reservedFixedIPsRaw))
+					for j, reservedFixedIP := range reservedFixedIPsRaw {
+						ipAddress := reservedFixedIP.(map[string]interface{})["ip_address"].(string)
+						results[j] = load_balancers.CreateStagedOptsReservedFixedIP{
+							IPAddress: ipAddress,
+						}
```

Comment:
[ASK]
`resourceMLBLoadBalancerV1Create` や変更前と同様に ↓ だとダメでしょうか?

```suggestion
						results[j] = load_balancers.CreateStagedOptsReservedFixedIP{
							IPAddress: reservedFixedIP.(map[string]interface{})["ip_address"].(string),
						}
```

Reply:
おっしゃる通り、不必要な変数化だったので直接入れ込むように修正しました